worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -510,16 +510,24 @@ private[celeborn] class Worker(
           // During graceful shutdown, to avoid allocate slots in this worker,
           // add this worker to master's blacklist. When restart, register worker will
           // make master remove this worker from blacklist.
-          rssHARetryClient.send(ReportWorkerUnavailable(List(workerInfo).asJava))
-          val interval = conf.checkSlotsFinishedInterval
-          val timeout = conf.checkSlotsFinishedTimeoutMs
-          var waitTimes = 0
-
-          def waitTime: Long = waitTimes * interval
+          try {
+            rssHARetryClient.askSync(
+              ReportWorkerUnavailable(List(workerInfo).asJava),
+              OneWayMessageResponse.getClass)
+          } catch {
+            case e: Throwable =>
+              logError(
+                s"Fail report to master, unreleased PartitionLocation: \n$partitionLocationInfo",
+                e)
+              return
```

Comment:
Don't think here should just return..., it will directly lose data. we can just ignore the exception...

Reply:
done..